website/static/static/js/version.js

```diff
@@ -1,5 +1,5 @@
 const presto_latest_presto_version = '0.224';
-const presto_latest_num_commits = 114;
-const presto_latest_num_contributors = 33;
-const presto_latest_num_committers = 14;
+const presto_latest_num_commits = 79;
+const presto_latest_num_contributors = 21;
+const presto_latest_num_committers = 10;
```

Comment:
How will contributors and committers values be presented?
If this gets showed on https://prestodb.github.io/ next to each other:
> Contributors | 33
> Committers | 14

it's suggestive that Committers are someone else than Contributors (ie 33+14=47 people were involved in the release). Is this the case?